tests/api_connexion/endpoints/test_dag_source_endpoint.py

```diff
@@ -40,7 +50,83 @@ def setUpClass(cls) -> None:
     def setUp(self) -> None:
         self.client = self.app.test_client()  # type:ignore
 
-    @pytest.mark.skip(reason="Not implemented yet")
-    def test_should_response_200(self):
-        response = self.client.get("/api/v1/health")
-        assert response.status_code == 200
+    @staticmethod
+    def _get_dag_file_docstring(fileloc: str) -> str:
+        with open(fileloc) as f:
+            file_contents = f.read()
+        module = ast.parse(file_contents)
+        docstring = ast.get_docstring(module)
+        return docstring
+
+    @parameterized.expand([("True",), ("False",)])
+    def test_should_response_200_text(self, store_dag_code):
+        serializer = URLSafeSerializer(conf.get('webserver', 'SECRET_KEY'))
+        with conf_vars(
+            {("core", "store_serialized_dags"): store_dag_code}
+        ):
+            dagbag = DagBag(dag_folder=EXAMPLE_DAG_FILE)
+            dagbag.sync_to_db()
```

Comment:
Does this create side effect on db level? Should we add setup/teardown?

Reply:
It may the case. Do you think it is good idea to use `tests.test_utils.db.clear_db_dags` here?

Reply:
Yup, I would add this in setup and teardown

Reply:
Done. I also changed mocking of the `STORE_DAG_CODE` in configuration since it is read during initial import.